pkg/resolve/resolve_test.go

```diff
@@ -139,6 +144,12 @@ func TestPipelineRunWithFinally(t *testing.T) {
 	assert.Equal(t, resolved.Spec.PipelineSpec.Finally[0].TaskSpec.Steps[0].Name, "finally-task")
 }
 
+func TestPipelineRunWithFinallyV1(t *testing.T) {
+	resolved, _, err := readTDfile(t, "pipeline-finally-v1", false, true)
+	assert.NilError(t, err)
+	assert.Equal(t, resolved.Spec.PipelineSpec.Finally[0].TaskSpec.Steps[0].Name, "finally-task")
+}
```

Comment:
If i understand correctly this test is not related to resolve right ?

Reply:
yeah just added them along the way while reading the code to make sure the code is  robust, i can split it if needed but it just tests....

Reply:
i was increasing the coverage as well..

Reply:
Oh okok
I think that's fine to keep here